docs/guides/src/main/server/configuring-database.adoc

```diff
@@ -0,0 +1,126 @@
+<#import "/templates/guide.adoc" as tmpl>
+<#import "/templates/kc.adoc" as kc>
+<#import "/templates/options.adoc" as opts>
+
+<@tmpl.guide
+    title="Configuring a database"
+    summary="An overview about how to configure relational databases"
+    includedOptions="db db-*">
+
+In this guide, you are going to understand how to configure the server to store data using different relational databases. You should also learn
+what are the databases supported by the server and how to proceed when using a unsupported database version.
```

Comment:
```suggestion
what are databases are supported by the server.
```

Reply:
another suggestion, without doubled "are" as in @stianst s ;) : 
```suggestion
You will also learn
what databases are supported by the server.
```